index.html

```diff
@@ -1004,6 +1006,103 @@ <h3>
           conventions or limitations of the host operating system.
         </p>
       </section>
+      <section>
+        <h3>
+          `protocol_handlers` member
+        </h3>
+        <p>
+          The [=manifest's=] <code><dfn data-export="" data-dfn-for=
+          "manifest">protocol_handlers</dfn></code> member is an <a>array</a>
+          of <a>protocol handler description</a>s that allows a web application to
+          handle URL protocols.
+        </p>
+        <p class="note">
```

Comment:
This sounds like an example... 

```suggestion
        <p class="example">
```

Given that we have the example below, do we need this one? Could we combine them?